src/vonage/_internal.py

```diff
@@ -13,10 +14,13 @@
 
 
 class BasicAuthenticatedServer(object):
-    def __init__(self, host, user_agent, api_key, api_secret, timeout=None):
+    def __init__(self, host, user_agent, api_key, api_secret, timeout=None, pool_connections=10, pool_maxsize=10, max_retries=3):
```

Comment:
Hello @maxkahan , I have set them to 10 and 10 because indeed they are the default values, but having them in the method signature allows anyone to change the default values and apply them to the Vonage Session object. If we remove them from the method signature there will be no way to change these default values when initiating a vonage client